frontend/src/css/style.css

```diff
@@ -562,6 +562,70 @@ button:disabled {
     outline: none;
 }
 
+.question-header {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.question-header-info {
+    display: flex;
+    flex-flow: column nowrap;
+    font-size: 1.2em;
+}
+
+.question-header-info-box {
+    border-radius: 10px;
+    text-align: center;
+    padding: 5px 65px;
+}
+
+.question-header-info-box:nth-child(2)
+{
+    margin-top: 10px;
+}
```

Comment:
Maybe add a text-transform: capitalize; To ensure the uppercase of the first letter of the question type?

Reply:
Good idea